doc/release-notes.md

```diff
@@ -1,65 +1,50 @@
 # Changes to `ga4gh/schemas` `master` branch since version `v0.5.1` (Oct 2, 2014)
 
-### Pervasive changes
+## Pervasive changes
 
 Renamed all protocols and records to remove the "`GA`" prefix from
 their names.  (Hence `GASearchVariantSetsRequest` is now
-`SearchVariantSetsRequest`, etc.)  The exception is `GAException`,
-which is unchanged.
+`SearchVariantSetsRequest`, etc.)  
+The exception is `GAException`, which is unchanged.
 
 There is no longer one unitary namespace.  Objects now reside in
 "`org.ga4gh.models`", methods in "`org.ga4gh.methods`".
 
-Updated the version number to `0.6.0-SNAPSHOT`.
+Updated the version number to `0.6.a`.
```

Comment:
What is the version number? I would vote for following [PEP 440](https://www.python.org/dev/peps/pep-0440/) and using either '0.6a1' or '0.6.0a1', depending on how many digits of precision we want.